src/client/browser/index.js

```diff
@@ -46,7 +46,7 @@ export function checkStatus (statusUrl, XMLHttpRequest) {
                 document.location = res.url;
 
             //NOTE: prepare to close the browser
-            if (res.cmd === COMMAND.close)
+            if (res.cmd === COMMAND.idle)
```

Comment:
set the document.title when page is loaded. `COMMAND.idle` is checked in the previous `if` statement